src/potions.ts

```diff
@@ -460,18 +484,64 @@ const rufusPotion = new Potion($item`closed-circuit pay phone`, {
   },
 });
 
-export const wishPotions = Effect.all()
-  .filter((effect) => !effect.attributes.includes("nohookah"))
+export const wishPotions = validWishes.map(
+  (effect) =>
+    new Potion($item`pocket wish`, {
+      effect,
+      canDouble: false,
+      duration: 20,
+      use: (quantity: number) =>
+        new Array(quantity).fill(0).every(() => {
+          const s = getValidWishString(effect);
+          if (s === "") return;
+          cliExecute(`genie effect ${s}`);
+        }),
+    })
+);
+
+export const pawPotions = validWishes
   .map(
     (effect) =>
-      new Potion($item`pocket wish`, {
+      // eslint-disable-next-line libram/verify-constants
+      new Potion($item`cursed monkey's paw`, {
         effect,
         canDouble: false,
-        duration: 20,
-        use: (quantity: number) =>
-          new Array(quantity).fill(0).every(() => cliExecute(`genie effect ${effect}`)),
+        price: () =>
+          get("_pawWishes", 0) >= 5 ||
+          // eslint-disable-next-line libram/verify-constants
+          !have($item`cursed monkey's paw`) ||
+          failedWishes.includes(effect)
+            ? Infinity
```

Comment:
I love using `Infinity`, but keep in mind that if we ever `sort` the array it'll cause issues (the `reduce` used in `maxBy` is fine, though)

Reply:
I think it's fine. We should never want to use a potion that has a cost of Infinity